index.html

```diff
@@ -142,17 +142,20 @@ <h2>Skills</h2>
       <div class="modal" id="modal-id"></div>
     </div>
   </main>
-    <footer id="contact-me">
+    <footer id="contact-me-footer">
       <div class="footer-wrapper">
         <div class="chat-text">
           <h2>
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
           </h2>
         </div>
-        <form action="https://formspree.io/f/xoqrojkw" method="POST">
+        <form id="contact-me" action="https://formspree.io/f/xoqrojkw" method="POST">
           <div class="input-field">
-            <input type="text" name="name" maxlength="30" class="text-input" placeholder="Full name" required>
-            <input type="email" name="Email" class="text-input" placeholder="Email address" required>
+            <input type="text" name="name" id="user-name" maxlength="30" class="text-input" placeholder="Full name" required>
+            <input type="email" id="user-email" name="Email" class="text-input" placeholder="Email address" required>
```

Comment:
- [ ] Kindly fix this
-  If a user submits the contact form with the email field containing uppercase characters an error message should show up.

Reply:
I have already added the error message when the email field has uppercase characters on line 156. can you kindly please check the code.